src/System.Private.ServiceModel/tools/test/Bridge/WcfTestBridgeCommon/CertificateGenerator.cs

```diff
@@ -103,7 +103,7 @@ public void Initialize()
                 Trace.WriteLine(string.Format("    {0} = {1}", "ValidityPeriod", _validityPeriod));
                 Trace.WriteLine(string.Format("    {0} = {1}", "Valid to", _validityNotAfter));
 
-                _authorityCertificate = CreateCertificate(true, false, null, string.Empty);
+                _authorityCertificate = CreateCertificate(true, false, null, null);
```

Comment:
Yeah, good suggestion @roncain.

@hongdai: Parameter naming syntax is something like: 

```csharp
_authorityCertificate = CreateCertificate(isAuthority: true, isMachineCert: false, signingCertificate: null, certificateCreationSettings: null);
```

basically naming the parameters as you pass them in